### PR TITLE
Display a loading message instead of blank page

### PR DESCRIPTION
**Pull request by xPaw**
07-1755-f271e11

### CODE REVIEW

**Comment by maxpoulin64** [comment]
I'm not sure about this one. I think having a loading screen is a good idea, but it would be nice if it reflected the correct status so it doesn't talk about Javascript and connectivity issues when it's fine. Even on localhost it takes a few seconds to load, and it's not rare for it to take 10-15 seconds on mobile to load. I don't think we can have the progress, but just displaying what it's doing would be great.

**Comment by xPaw** [comment]
@maxpoulin64 Is that better? It will now say `Connecting` and `Authorizing` while hiding the message about javascript as soon as it loads.

**Comment by astorije** [comment]
I'll try this before giving a +1.

**Comment by maxpoulin64** [comment]
Love it, 👍

**Comment by xPaw** [comment]
I've rebased this and also fixed colors in morning/zenburn themes.

**Comment by JustRamon** [comment]
It works. Cool :)

**Comment by astorije** [comment]
This is really cool!
A few suggestions/comments before I'm a +1:

- I'm not sure about that `You must use a modern browser with JavaScript enabled.` statement. If it stays on for too long, it suggests the user that they do not have a compatible browser. At the very least, I would change it for `Make sure to use a modern browser with JavaScript enabled.` so that it shows we do not check that ourselves, but personally I'd rather ditch it entirely.
- What about a timeout to display `If this message still does not go away, there might be connectivity issues.`? There is no need to display it the first, say 5 seconds where this is loading so having it appear only after some time makes sense to me.
- `You're not connected to any networks yet` displays when the page is loading, which has always bugged me. This would be a good time to hide it until the page is loaded (and the user is authenticated in private mode <sup>1</sup>).

<sup>1</sup> It would actually be nicer to disable the sidebar menu on the login page, but that's a different PR :-)

File: client/index.html [hunk 1]
```diff
@@ -43,7 +43,22 @@
...
+									<h2 class="title">The Lounge is loading…</h2>
+								</div>
+								<div class="col-xs-12">
+									<p id="loading-page-message">Loading the app… <a href="http://enable-javascript.com/" target="_blank">Make sure to have JavaScript enabled.</a></p>
+									<p id="loading-slow" style="display:none">This is taking longer than it should, there might be connectivity issues.</p>
+								</div>
+							</div>
+						</div>
+					</div>
+					<script async src="js/loader.js"></script>
```
Comment (astorije) [inline client/index.html]: Why `async` here?

Comment (xPaw) [inline client/index.html]: This script is not important enough to block for it.

Comment (astorije) [inline client/index.html]: Would call it loading-slow-check.js or something like that, unless you are planning to add more loader business there.

**Comment by astorije** [comment]
This is great, 👍 and merging!

> `You're not connected to any networks yet` displays when the page is loading, which has always bugged me. This would be a good time to hide it until the page is loaded (and the user is authenticated in private mode<sup>1</sup>).

Not addressed but can definitely be part of a further PR, this is already a great step ;-)

**Pull request merged**